src/components/encoding-pane/encoding-shelf.tsx

```diff
@@ -134,6 +137,7 @@ class EncodingShelfBase extends React.PureComponent<EncodingShelfProps, {}> {
       <div styleName='field-wrapper'>
         <Field
           draggable={true}
+          filters={filters}
```

Comment:
Why does field inside encoding-shelf needs an array of filter?  This seems like an overkill.